gradle-plugin/src/main/kotlin/com/google/devtools/ksp/gradle/KspConfigurations.kt

```diff
@@ -15,6 +15,14 @@ class KspConfigurations(private val project: Project) {
         private const val PREFIX = "ksp"
     }
 
+    private val allowAllTargetConfiguration =
+        project.findProperty("ksp.allow_all_target_configuration")?.let {
```

Comment:
nit: Maybe `ksp.allow.all.target.configuration` or `ksp.allowAllTargetConfiguration` to match Gradle/Kotlin/AGP naming.